adafruit_mcp9808.py

```diff
@@ -74,26 +107,25 @@ class MCP9808:
 
     """
 
-    # alert_lower_temperature_bound
-    # alert_upper_temperature_bound
-    # critical_temperature
-    # temperature
-    # temperature_resolution
+    _MCP9808_REG_RESOLUTION_SET = RWBits(2, 0x08, 0, register_width=2)
+    above_ct = ROBit(_MCP9808_REG__TEMP, 7, register_width=1)
+    above_ut = ROBit(_MCP9808_REG__TEMP, 6, register_width=1)
+    below_lt = ROBit(_MCP9808_REG__TEMP, 5, register_width=1)
```

Comment:
You want comments after these so they show up in sphinx. For example:
`True when the temperature is higher than the currently set critical temperature. False otherwise.`

Reply:
Will do Thanks

Reply:
done in https://github.com/adafruit/Adafruit_CircuitPython_MCP9808/pull/29/commits/829bc12cea1871467b0ad22e1fb9c92dcf21885a